app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,35 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserLoginDetail;
+import com.codesoom.assignment.errors.AuthenticationFailException;
+import com.codesoom.assignment.utils.JwtUtil;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+/**
+ * 사용자 인증 서비스
```

Comment:
이렇게 표현할 수도 있겠죠? 어떤가요?
```suggestion
 * 사용자 인증 관련 기능을 제공합니다.
```

Reply:
네! 좋습니다